src/libraries/IntegrationTests/tests/iOS/iOS.Integration.Tests.csproj

```diff
@@ -0,0 +1,88 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The name needs to be better. Is this device or simulator? Is this interpreter, aot, mixed or what mode?

Reply:
I guess the original sample is about AOT on simulator. So the name could be  `src/libraries/IntegrationTests/tests/iOS/iOS.Simulator.Aot.Integration.Tests.csproj`

Reply:
It was either or as `$(UseAotForSimulator)` made that determination.

Reply:
Done